src/main/java/org/folio/edgecommonspring/util/ApiKeyUtils.java

```diff
@@ -0,0 +1,52 @@
+package org.folio.edgecommonspring.util;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import java.util.Base64;
+import org.folio.edge.api.utils.model.ClientInfo;
+
+public class ApiKeyUtils {
```

Comment:
The same as https://github.com/folio-org/edge-common-spring/pull/2/files#r601843747. Can this be the common functionality?

Reply:
ApiKeyUtils in the edge-common use `io.vertx.core.json.JsonObject` for spring way we use `ObjectMapper`
Please, double check: https://github.com/folio-org/edge-common/blob/master/src/main/java/org/folio/edge/core/utils/ApiKeyUtils.java

Reply:
I agree this belongs in edge-api-utils.  When edge-common adopts edge-api-utils it can make the necessary json library adjustments

Reply:
will create a PR in edge-api-utils